libs/books/feature/src/lib/book-search/book-grid/book-grid.component.html

```diff
@@ -0,0 +1,3 @@
+<ng-container *ngIf="books?.length">
+  <tmo-book-grid-item data-testing="book-item" *ngFor="let b of books" [book]=b></tmo-book-grid-item>
```

Comment:
Do you really need the `ng-container` here? What will the `ngFor` output if `books` is null or empty?

Reply:
adding `empty` block for when there are no books found

Reply:
This isn't quite what I meant.

Why couldn't you simply reduce this entire component to just line 2?